examples/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/fullstorydev/emulators/examples
 
-go 1.18
+go 1.21
+
+toolchain go1.21.6
```

Comment:
Must we enforce a Go version higher than the one required by the actual emulators packages?
Currently they require only `go 1.18`.

Reply:
Actually this seems to be what's breaking CI for Go versions < 1.21.

Reply:
oh!  that's annoying.. it just did that automatically :/
I'll revert.